docs/streams/stream-examples/realtime-updates.md

```diff
@@ -1,33 +1,40 @@
 ---
 sidebar_position: 140
-title: Realtime DB Updates Example
+title: Real-Time DB Updates
```

Comment:
Removing "Example" from the title breaks the pattern of titles in this section. Please revert this change or we can discuss removing "Example" from the other titles.